### ISSUE TITLE
With xdist: allow to decide the minimal size of chunks for the slaves

### ISSUE THREAD

**Issue by pytestbot** [pytest-dev/pytest#738]
Originally reported by: **BitBucket: [teudimundo](http://bitbucket.org/teudimundo), GitHub: [teudimundo](http://github.com/teudimundo)**

----------------------------------------

In a specific [scenario](http://stackoverflow.com/questions/30058216/pytest-xdist-running-1-test-per-slave), I would need all the tests to be run in parallel at the same time. 

Unfortunately seems that at the moment each slave will be fed with chunk of a minimal size of 2 tests.

I don't know if it might work but an argument to decide the size (or the minimal size) of chunks can be given. A further possibility would be to have a new value for -n could mean to allocate the number of slaves needed to run all the chunks in parallel.

So that using

py.test -n by-chunk-size --chunk-size 1

would execute all of them in parallel.
 

----------------------------------------
- Bitbucket: https://bitbucket.org/pytest-dev/pytest/issue/738

**Comment by RonnyPfannschmidt** [comment]
Superseded by pytest-xdist issues
